yarn-project/pxe/src/pxe_oracle_interface/pxe_oracle_interface.ts

```diff
@@ -732,17 +732,26 @@ export class PXEOracleInterface implements ExecutionDataProvider {
     return new LogWithTxData(trimmedLog, scopedLog.txHash, txEffect.data.noteHashes, txEffect.data.nullifiers[0]);
   }
 
-  // TODO(#12553): nuke this as part of tackling that issue. This function is no longer unit tested as I had to remove
-  // it from pxe_oracle_interface.test.ts when moving decryption to Noir (at that point we could not get a hold of
-  // the decrypted note in the test as TS decryption no longer existed).
```

Comment:
My comment. In hindsight I think I was just lazy updating the test. Shame on me.

Added the test in a PR up the stack.